frontend/src/components/Survey/CreateSurvey.js

```diff
@@ -1,59 +1,369 @@
+                                        return (
+                                            <Box key={i} className="list-question list-question-flush">
+                                                <Box className="list-question-item">
+                                                    <Typography version='h5' className="card-title" >Questionaire #{i + 1} </Typography>
+                                                    <Box className="form-row">
+                                                        <Box className="form-question col-6">
+                                                            <OutlinedInput 
+                                                                name={`participants_emails.${i}.type1_question`} 
+                                                                type="text" 
+                                                                className={'form-control' + (ticketErrors.email && ticketTouched.email ? ' is-invalid' : '' )}
+                                                            />
+                                                            <ErrorMessage name={`participants_emails.${i}.type1_question`} component="Box" className="invalid-feedback" />
+                                                        </Box>
+                                                        <Box className="form-question col-6">
+                                                            <OutlinedInput 
+                                                                name={`participants_emails.${i}.type2_question`} 
+                                                                type="text" 
+                                                                className={'form-control' + (ticketErrors.email && ticketTouched.email ? ' is-invalid' : '' )}
+                                                            />
+                                                            <ErrorMessage name={`participants_emails.${i}.type2_question`} component="Box" className="invalid-feedback" />
+                                                        </Box>
+                                                    </Box>
+                                                </Box>
+                                            </Box>
+                                        );
+                                    }))}
+                                </FieldArray>
+                                </Box>
+                                </Grid>
+                            </Grid>
+                        </Box>
+                        <Button
+                            type="submit"
+                            fullWidth
+                            size="large"
+                            variant="contained"
+                            sx={{ mt: 3, mb: 2 }}
+                            onClick={onSubmit(values)}
```

Comment:
change this to `onClick={() => onSubmit(values)`. The way it is right now, it randomly calls the onSubmit function without even clicking the button